test/puppetlabs/trapperkeeper/authorization/testutils.clj

```diff
@@ -26,3 +31,9 @@
    (new-rule type path :any))
   ([type path method]
    (rules/new-rule type path method 500 "test rule")))
+
+(defn url-encoded-cert
+  [x509-cert]
+  (let [cert-writer (StringWriter.)
+        _ (ssl-utils/cert->pem! x509-cert cert-writer)]
+    (ring-codec/url-encode cert-writer)))
```

Comment:
Yeah, that'd probably be a good thing to do with those functions at some point, I agree.  Maybe the next time we're in ssl-utils and happen to think about it.  Don't think I'd bother for now.